cmd/zed/lake/ztests/commit-by-hash.yaml

```
@@ -0,0 +1,17 @@
+script: |
+  export ZED_LAKE_ROOT=test
+  zed lake init -q
+  zed lake create -q -p POOL
+  zed lake add -q -p POOL in.zson
+  hex=`zed lake status -p POOL -f zng | zq -f text "cut s='0x'+to_hex(id)" -`
```

We usually use `$()` for command substitution.
```suggestion
  hex=$(zed lake status -p POOL -f zng | zq -f text "cut s='0x'+to_hex(id)" -)
```